### DISCUSSION TITLE
Having the Server paste in Dupes

### DISCUSSION THREAD

**Discussion by C-olossal** [wiremod/advdupe2#461] [Q&A] [answered]
I want my server to paste in props for an event that happens every hour or so automatically. Is this something adv dupe is currently capable of doing without extreme modifications?

**Comment by thegrb93** [comment, marked as answer]
You can add a hook here to call PasteMap. https://github.com/wiremod/advdupe2/blob/master/lua/autorun/server/advdupe2_sv_init.lua#L117-L118

Pretty easy
